sirepo/feature_config.py

```diff
@@ -130,6 +130,8 @@ def b(msg, dev=False):
         ),
         sim_common=dict(
             hide_guest_warning=b('Hide the guest warning in the UI', dev=True),
+            join_slack_uri=('https://slack.sirepo.com/v1/', str,
```

Comment:
Let's not expose this in the source code. Rather, just have `https://slack.com/` as the value in the default configuration. We'll set the configuration on production.

Reply:
1e63e3e61d6b75f5c183715519625d2fdca71b42